### DISCUSSION TITLE
kittens.tui.handler could not be resolved

### DISCUSSION THREAD

**Discussion by dpetka2001** [kovidgoyal/kitty#4203] [Q&A] [answered]
I installed kitty with getting the installer by curl and then running it, since my package manager has a rather old version (0.15 on Linux Mint). I'm trying to run a kitten, namely the zoom_toggle.py, but when i copy/paste the code from the web page to my editor, I get an error "kittens.tui.handler" could not be resolved. Is there a way to solve this? Thnx in advance.

**Comment by kovidgoyal** [comment, marked as answer]
You probably have two versions of kitty on your system. Remove the
distro one. And in anycase there is no need for zoom_toggle any more its
built in. https://sw.kovidgoyal.net/kitty/actions/#toggle-layout

**Reply by dpetka2001** [reply]
I don't have 2 versions of kitty installed. Only the one by curl and running the installer. Still the toggle-layout option 
fits my needs and will go on using that. Thnx for your help.
